### ISSUE TITLE
Question about start training with CIFAR-100 or ImageNet (32x32)

### ISSUE THREAD

**Issue by headscott** [poloclub/robust-principles#4]
I am new to all that AI training. And I want to train your model with 

`make experiments/RaResNet50/.done_train`

First, I got access to ImageNet. But which of all of these datasets are supposed to be used for the training? Is it "ImageNet Large-scale Visual Recognition Challenge (ILSVRC) from 2012"? and then these ones:

```
[Training images (Task 1 & 2)](https://image-net.org/data/ILSVRC/2012/ILSVRC2012_img_train.tar). 138GB

 [Training images (Task 3)](https://image-net.org/data/ILSVRC/2012/ILSVRC2012_img_train_t3.tar). 728MB

 [Validation images (all tasks)](https://image-net.org/data/ILSVRC/2012/ILSVRC2012_img_val.tar). 6.3GB.
```
?

These are pretty big files, so I wanted to use train(32x32) and val(32x32) or CIFAR-100 for my first try. But I can't find how to correctly setup these. If I use a script, where it puts alle the .jpg train files into /imagenet/train/{label}/{file}.jpg the command from your Makefile says "No such file or directory: '/imagenet/160/train'. If I use CIFAR-100 I didn't even find out where to get the val data for validation. I also tried to use the resize.py script from the 4th step on https://github.com/locuslab/fast_adversarial/tree/master/ImageNet which should not be needed according to your "Getting Started" step 1, because I don't want to run fast aversarial training. 

So what should I do? What am I doing wrong?

**Comment by headscott** [comment]
**Edit / Update**

I found the reason for my confusion.

It looks like I was using the wrong Make target in the first place. I initially ran:

```bash
make experiments/RaResNet50/.done_train
```

which expects the resized ImageNet datasets (`160` and `352`). That's why I thought I had to run `resize.py`.

However, if the intended experiment is the standard PGD adversarial training with the RaResNet50 model, the correct command appears to be:

```bash
make experiments/PGDAT_RaResNet50/.done_train
```

So the `resize.py` step I was asking about here was actually **not necessary for the experiment I wanted to run**. I had simply followed the wrong Make target from the Getting Started instructions.

This also explains the `/imagenet/160/train` error I was getting.

In other words, my original question about how to prepare `imagenet/160` and `imagenet/352` was caused by the incorrect command in the Getting Started instructions, rather than by a missing step in my ImageNet setup.

See Issue #5
